### PR TITLE
Add change tracking

### PR DESCRIPTION
**Pull request by vmikhailenko**
Adds change tracking for orders in the admin view.
Bumps Fiori version.

### CODE REVIEW

File: srv/admin-service.cds [hunk 1]
```diff
@@ -40,3 +43,25 @@ annotate AdminService.Books with @odata.draft.enabled;
...
+    book.isbn
+  ]
+};
+
+// Assign identifiers to the tracked entities
+annotate AdminService.Orders with @changelog: [OrderNo];
+annotate AdminService.OrderItems with @changelog: [
+    Items.parent.OrderNo,
+    Items.book.title,
+    Items.book.isbn
```
Comment (beckermarc) [inline srv/admin-service.cds:66]: Why prefix with `Items`?

File: srv/admin-service.cds [hunk 2]
```diff
@@ -40,3 +43,25 @@ annotate AdminService.Books with @odata.draft.enabled;
...
+// Change-track orders and items
+annotate AdminService.Orders {
+  OrderNo @changelog;
+};
+
+annotate AdminService.OrderItems {
+  amount @changelog;
+  quantity @changelog;
+  book @changelog: [
+    book.title,
```
Comment (beckermarc) [inline srv/admin-service.cds:56]: `book` prefix required?

File: srv/pom.xml [hunk 3]
```diff
@@ -72,6 +72,11 @@
 			<scope>runtime</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>com.sap.cds</groupId>
+			<artifactId>cds-feature-change-tracking</artifactId>
```
Comment (beckermarc) [inline srv/pom.xml:77]: Please add runtime scope for now

File: srv/admin-service.cds (hunk 2 above)
Comment (vmikhailenko) [reply srv/admin-service.cds:56]: No.

File: srv/admin-service.cds (hunk 1 above)
Comment (vmikhailenko) [reply srv/admin-service.cds]: Here - no.

File: integration-tests/pom.xml [hunk 4]
```diff
@@ -94,20 +94,27 @@
...
-							<arguments>install --no-save</arguments>
+							<arguments>run build</arguments>
 							<skip>${skipTests}</skip>
 							<workingDirectory>${sidecar.dir}</workingDirectory>
 						</configuration>
 					</execution>
 				</executions>
 			</plugin>
 
+			<!-- Introduces a hack: MTX sidecar is executed in the pseudo-production mode
```
Comment (beckermarc) [inline integration-tests/pom.xml:110]: There must be a better way, I think. Did you check if setting 

```
"cds": {
  "cdsc": {
    "moduleLookupDirectories": [ "node_modules/", "target/cds/" ]
  }
}
```

in mtx/sidecar/package.json would help? Maybe also only for development profile (not sure though why it works in production, but not in development profile, smells like a bug in MTX Sidecar)

File: mtx/sidecar/package.json [hunk 5]
```diff
@@ -18,6 +18,9 @@
     "profile": "mtx-sidecar",
     "[development]": {
       "requires": { "auth": "dummy" }
+    },
+    "cdsc": {
+      "moduleLookupDirectories": [ "node_modules/", "srv/target/cds/" ]
```
Comment (beckermarc) [inline mtx/sidecar/package.json:23]: Hm, this should ideally be the default "target/cds/". Doesn't this work?

**Review by beckermarc — APPROVED** [review]

**Pull request merged**